src/Symfony/Component/Translation/Translator.php

```diff
@@ -376,6 +376,10 @@ private function doLoadCatalogue($locale)
         if (isset($this->resources[$locale])) {
             foreach ($this->resources[$locale] as $resource) {
                 if (!isset($this->loaders[$resource[0]])) {
+                    if (\is_string($resource[1])) {
+                        throw new RuntimeException(sprintf('The "%s" translation loader which is needed to load the "%s" resource is not registered.', $resource[0], $resource[1]));
```

Comment:
does it even exist? to me the message suggests so
shouldn't the message say something like:
No loader is registered for the "twig" format, etc.?

Reply:
good idea, done